src/language-js/printer-estree.js

```diff
@@ -2833,6 +2833,8 @@ function printPathNoParens(path, options, print, args) {
       return "string";
     case "TSUndefinedKeyword":
       return "undefined";
+    case "TSUnknownKeyword":
+      return "unknown";
```

Comment:
Should we extract all of these “constant” nodes to an object to reduce the number of `case` statements needed?